### PR TITLE
DSFAAP-537: add email address page as first step in receive your lice…

### PR DESCRIPTION
**Pull request by joseluisgraa**
…nse task

### CODE REVIEW

File: src/server/common/model/answer/email-address.test.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+]
+const invalidEmailAddresses = [
+  'plainaddress',
+  '@no-local-part.com',
+  'no-at.domain.com'
+]
+
+describe('EmailAddress', () => {
+  describe('validate', () => {
+    it('should return true for valid email address', () => {
```
Comment (DrogoNevets) [inline src/server/common/model/answer/email-address.test.js:35]: move the `it` into the for - so it runs a seperate test for each email (this is consistent with how we did it for postcodes)

File: src/server/common/model/answer/email-address.test.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      const { isValid, errors } = emailAddress.validate()
+
+      expect(isValid).toBe(false)
+      expect(errors.emailAddress.text).toBe(
+        'Enter the email address you would like the licence sent to'
+      )
+    })
+
+    it('should return false for invalid email address', () => {
+      invalidEmailAddresses.forEach((email) => {
```
Comment (DrogoNevets) [inline src/server/common/model/answer/email-address.test.js:57]: as above, put the test into the loop

File: src/server/license/email-address/controller.test.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
+import { createServer } from '~/src/server/index.js'
+import { statusCodes } from '~/src/server/common/constants/status-codes.js'
+import { pageTitle } from './controller.js'
+import { withCsrfProtection } from '~/src/server/common/test-helpers/csrf.js'
+import { parseDocument } from '~/src/server/common/test-helpers/dom.js'
+
+describe('licenseEmailAddress', () => {
```
Comment (DrogoNevets) [inline src/server/license/email-address/controller.test.js:7]: no tests checking its been added to state correctly/incorrectly

**Review by DrogoNevets — CHANGES_REQUESTED** [review]

File: src/server/common/model/answer/email-address.test.js (hunk 1 above)
Comment (joseluisgraa) [reply src/server/common/model/answer/email-address.test.js]: done

File: src/server/common/model/answer/email-address.test.js (hunk 2 above)
Comment (joseluisgraa) [reply src/server/common/model/answer/email-address.test.js]: done

File: src/server/license/email-address/controller.test.js (hunk 3 above)
Comment (joseluisgraa) [reply src/server/license/email-address/controller.test.js:10]: Done!

**Review by DrogoNevets — APPROVED** [review]

**Pull request merged**